python/tk_multi_loader/model_publishhistory.py

```diff
@@ -186,10 +213,11 @@ def _populate_thumbnail_image(self, item, field, image, path):
         :param field: The Shotgun field which the thumbnail is associated with.
         :param path: A path on disk to the thumbnail. This is a file in jpeg format.
         """
-        if field == "image":
+        image_field = utils.get_item_image_field(item)
+        if image_field and field == image_field:
             thumb = QtGui.QPixmap.fromImage(image)
             item.setData(thumb, SgPublishHistoryModel.PUBLISH_THUMB_ROLE)
-        else:
+        elif field == "created_by.HumanUser.image":
```

Comment:
Strange to change the `else` with an `elif`... So what should happen for the `else`?

Reply:
The else for me was weirder...
The `populate_thumbnail_image` gets called automatically for all image fields that are loaded in the data if download thumbnails is set to True. Here, we changed the default implementation to only download interesting fields, but before our changes, it would download them all. With the `else`, it would mean that any image downloaded that was not the Published File's image could eventually end up being the user's avatar in the UI...

Reply:
Please add some comments explaining that then.